src/static/client/app/signup/signup.component.scss

```diff
@@ -1,110 +1,129 @@
 * {
-    box-sizing: border-box;
-    margin: 0;
-    padding: 0;
-  }
-  
-  body, html {
-    height: 100%;
-    font-family: Arial, sans-serif;
-  }
-  
-  .signup {
-    background: linear-gradient(to right, #1632aa, #B30D2C);
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body,
+html {
+  height: 100%;
+  font-family: Arial, sans-serif;
+  background-color: #fff; // Changed the background color to white
+}
+
+.signup {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+
+  .button-container {
     display: flex;
+    flex-direction: column;
     justify-content: center;
     align-items: center;
-    height: 100vh;
+  }
 
-    .button-container {
-      display: flex;
-      justify-content: space-between;
-    }
-    
-    #icon {
-      width: 150px;  // Adjust as needed to change the size of the logo.
-      height: auto;
-      display: block;
-      margin: 0 auto 20px auto; // This centers the image and adds space below it.
+  #icon {
+    width: 60px; // Adjust as needed to change the size of the logo.
+    height: auto;
+    display: block;
+    margin: 0 auto 20px auto; // This centers the image and adds space below it.
+  }
+
+  .wrapper {
+    max-width: 400px;
+    background-color: #fff;
+    border-radius: 10px;
+    box-shadow: 0 4px 6px rgba(0, 0, 0, 0.1), 0 1px 3px rgba(0, 0, 0, 0.08);
+    padding: 30px;
+    width: 100%;
+    max-width: 350px;
+
+    .h2 {
+      text-align: center;
+      margin-bottom: 24px;
     }
-  
-    .wrapper {
-      max-width: 400px;
-      background-color: #fff;
-      border-radius: 10px;
-      box-shadow: 0 4px 6px rgba(0, 0, 0, 0.1), 0 1px 3px rgba(0, 0, 0, 0.08);
-      padding: 30px;
-      width: 100%;
-      max-width: 350px;
-  
-      .h2 {
-        text-align: center;
-        margin-bottom: 24px;
-      }
-  
-      #formContent {
-        display: flex;
-        flex-direction: column;
-  
-        .form-label {
-          display: block;
-          font-size: 14px;
-          font-weight: 600;
-          color: #4a4a4a;
+
+    #formContent {
+      display: flex;
+      flex-direction: column;
+
+      .fadeIn.first {
+        margin-top: 30px; // Added to give some space above the logo
+
+        .info {
+          font-size: 10px; // Decrease the font size for the instructions
```

Comment:
Usually font size shouldn't go below 16px for accessibility, but definitely not as low as 10px. I recommend boosting it to at least 14px.

Reply:
I agree with your point about the font size. Ensuring accessibility is important, and I should have been more mindful of that. I'll change the font size to 14px as per your recommendation. Thank you for bringing this to my attention.

Reply:
the font size is still 10px?